examples/nuxt/pages/index.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div id="view">
+    First name: {{ firstName }}
+    <br />
+    Last name: {{ lastName }}
+    <br />
+    server: {{ isServer ? 'true' : 'false' }}
+  </div>
+</template>
+
+<script>
+// I'd like to see how vue devs would refactor this, as I am a React dev
```

Comment:
I'm more of a React dev as well, but yea it doesn't seem right to have 3 computed props.

Maybe we can have two instead? One for `person` (so `person.firstName` instead of `firstName`) and another for `isServer`?